keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/plugins/VerificationEvaluator.kt

```diff
@@ -0,0 +1,12 @@
+package com.netflix.spinnaker.keel.api.plugins
+
+import com.netflix.spinnaker.keel.api.Verification
+import com.netflix.spinnaker.keel.api.verification.VerificationStatus
+
+interface VerificationEvaluator<VERIFICATION: Verification> {
+  val supportedVerification: Pair<String, Class<out Verification>>
```

Comment:
This should be using the generic type, right?
```suggestion
  val supportedVerification: Pair<String, Class<out VERIFICATION>>
```

Reply:
Hmm, I think that should actually just be `Class<VERIFICATION>`